src/server/util/kubectl.cc

```diff
@@ -118,7 +162,9 @@ static std::string generate_global_object(
                     "\n  name: " + object_id +
                     "\n  namespace: " + namespace_ +
                     "\n  labels:"
-                    "\n    job: none"
+                    "\n    job: " + job_name +
+                    "\n    created-by-podname: " + client_pod_name +
+                    "\n    created-by-podnamespace: " + client_pod_namespace +
```

Comment:
Adding such field to global object has no benefits, as we would never care where a global object is created.

Reply:
In an assembly operation, I think we need to get the original global object's metadata and build a new global object, the label is important in that scenario.